library_generation/generate_composed_library.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# This script allows generation of libraries that are composed of more than one
+# service version. It is achieved by calling `generate_library.sh` without
+# postprocessing for all service versions and then calling
+# postprocess_library.sh at the end, once all libraries are ready.
+#
+# Arguments
+# --generation_queries: a single string of key-value groups separated by a
+# pipe |. They key-value groups are in the form of `key=value` and will
+# be converted to an argument to generate_library.sh (`--key value`).
+#   example: "proto_path=google/cloud/asset/v1,destination_path=google-cloud-asset-v1-java,(...)|proto_path=google/cloud/asset/v1p2beta5,destination_path=google-cloud-asset-v1-java,(...)"
+#   In this case, generate_library.sh will be called once with
+#     generate_library.sh --proto_path google/cloud/asset/v1 --destination_path google-cloud-asset-v1-java
+#     and once with
+#     generate_library.sh --proto_path google/cloud/asset/v1p2bet5 --destination_path google-cloud-asset-v1p2beta5-java
+# --versions_file: a file of newline-separated version strings in the form "module:released-version:current-version". The versions will be applied to the pom.xml files and readmes
+#
+# Prerequisites
+# - Needs an `output` folder at the location of the calling shell. This folder
+# is automatically detected by `generate_library.sh` and this script ensures it
+# contains the necessary folders and files, specifically:
+#   - A "google" folder found in the googleapis repository
+#   - A "grafeas" folder found in the googleapis repository
+#   - A "WORKSPACE" file belonging to googleapis
```

Comment:
Why `WORKSPACE` is needed?

Reply:
Ah, I was initially trying to infer some options from the WORKSPACE, right before we moved to the `--generation_queries` approach. Thanks for the catch. I removed it.